[docs] New homepage; reorg docs; add start of eng design docs (#1131)
WorldHealthOrganization/app: docs/README.md

| How can I help out?                                      | We are humbled at the outpouring of support we have received from all over the world and are eager to have you contribute and apologize that we are still in the early process of figuring out how to manage all the contributions. Please bear with us while we get an initial version out the door and split out into smaller teams working in parallel, when we can then on-board more volunteers. Until then, please keep filing issues, and fill out the [volunteer form here](https://forms.gle/FUugWvUVvMcV3dLJA) and we’ll get back in touch. You can then read the [onboarding documentation](ONBOARDING.md). |

> | There's a problem with the app - help!                   | File a report to help us improve. Choose "Bug Report" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |

Reviewer: Link directly to Bug Report template?

```suggestion
| There's a problem with the app - help!                   | File a new bug report [here](https://github.com/WorldHealthOrganization/app/issues/new?template=bug-report.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Author: We can't do that.  The link doesn't just have the bug report template it, it includes the labels and other content which change over time.  No idea why GitHub does it that way :(

> | I have an idea for the app!                              | Choose "Feature Request" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |

Reviewer: Link directly to Feature Request template?

```suggestion
| I have an idea for the app!                              | Request a feature [here](https://github.com/WorldHealthOrganization/app/issues/new?template=feature-request.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Author: We can't do that.  The link doesn't just have the bug report template it, it includes the labels and other content which change over time.  No idea why GitHub does it that way :(

| How do I build the app myself?                           | See the [development guide](ONBOARDING.md#development).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

> | Who can I talk to about product, engineering, or design? | Reach out to our [functional leads](ONBOARDING.md#point-people).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |

Reviewer: I'm fine with this, but we are signing ourselves up for a lot of emails to filter through in this case... should we work with PgMs to think through a better place to direct people? Google Group maybe?

Author: I think we should do that, but at this moment we don't have that process.  Once we have that process, we can change this.
